test/draw/gpu/GpuImageScalingTest.ooc

```diff
@@ -0,0 +1,40 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+use base
+use geometry
+use draw-gpu
+use draw
+use opengl
+use unit
+
+GpuImageScalingTest: class extends Fixture {
+	init: func {
+		super("GpuImageScalingTest")
+		sourceImage := RasterRgba open("test/draw/gpu/input/Flower.png")
+		this add("Scaling X rotation", func {
+			correctImage := RasterRgba open("test/draw/gpu/correct/scaling_X_rotation.png")
+			gpuImage := gpuContext createRgba(IntVector2D new(200, 150))
+			gpuImage canvas fill(ColorRgba transparent)
+			DrawState new(gpuImage) setFocalLengthNormalized(0.1f) setTransformReference(FloatTransform3D createRotationX(5.0f toRadians())) setInputImage(sourceImage) draw()
+			rasterFromGpu := gpuImage toRaster()
+			expect(rasterFromGpu distance(correctImage), is equal to(0.0f))
```

Comment:
We've historically been bad at memory management in tests.
The fact that other tests are bad doesn't mean this new test has to be. Add the free.